templates/default/sql/dml/3.3.0.sql.erb

```diff
@@ -1,3 +1,7 @@
 REPLACE INTO `hopsworks`.`variables`(`id`, `value`, `visibility`) VALUES ("docker_cgroup_parent", "<%= node['hops']['docker']['cgroup']['parent'] %>", 1);
 REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES('unix_usernames_conf', '<%= @usernames_configuration %>');
 REPLACE INTO `hopsworks`.`variables`(`id`, `value`, `visibility`) VALUES ("loadbalancer_external_domain", "<%= node['hopsworks']['loadbalancer_external_domain'] %>", 1);
+
+REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("jupyter_remote_fs_driver", "<%= node['hopsworks']['jupyter']['remote_fs_driver'] %>");
+REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("hopsfsmount_apparmor_profile", "<%= node['hops']['docker']['hopfsmount-apparmor-profile'] %>");
+REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("enable_hopsfsmount_apparmor_profile_k8s", "<%= node['kube-hops']['fuse']['enable_apparmor_profile'] %>");
```

Comment:
What does it mean to enable apparmor? Apparmor is always enabled but you can use different profiles.

Also, please do not use abbreviations such as `k8s`, we already have `kubernetes` and `kube`.

Reply:
I mean to use the hopsfsmount_apparmor_profile or not. If it is false then we are going to disable AppArmor on the container.

Reply:
You don't disable apparmor, you are using different profiles. As there is no documentation, if we are not careful with the naming of the parameters in 6 months we will not know what is this parameter doing.